### DISCUSSION TITLE
Hope to get example code

### DISCUSSION THREAD

**Discussion by chc7042** [dutchigor/pluggable-electron#6] [General]
pluggable architecture is very helpful and powerful for our project. first of all thank you for your distribution.
i have one discussion point. is it possiible to provide example code for pluggable module
i am beginer of electron and more javascript so it really hard to implement. please help me

**Comment by dutchigor** [comment]
Good point. I'm currently working on an update that provides support for modern frameworks like React and Vue. This will change how Pluggable Electron should be used somewhat and make it a bit simpler, so I'll get this done first and then provide an example starter project along with that.

In the mean time, the home page on the wiki already includes all the code needed to get this set up. See the file that the code goes in to in the 1st comment of each code block. But this still needs to be inserted into an existing project and I undestand that that can get a bit confusing so a template definitely makes sense.

**Reply by chc7042** [reply]
based on your example i will follow instruction but i think i need to get some module installation and entry point of plugin. so i imagine the way to solve this but it is diffcult to me. because of this i think example code very helpful to understand what pluggable works. i really hope to get example code including menu

**Comment by dutchigor** [comment]
The new version is now available with updated instructions on how to use it. I recommend using the new vesion as this works with ES modules so it can be used with modern frameworks. 
An example using plain javascript is now available in <https://github.com/dutchigor/pluggable-electron-demo>. I hope to get round to preparing an example using Vue and React as well but that will have to come after my holiday.

**Reply by dutchigor** [reply]
The above mentioned library is now updated with a branch containing an example using Vue. See https://github.com/dutchigor/pluggable-electron-demo/tree/with-vue
